rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -309,6 +309,11 @@ error_code sys_rsx_context_iomap(u32 context_id, u32 io, u32 ea, u32 size, u64 f
 		return CELL_EINVAL;
 	}
 
+	if (render->ctrl->get != (render->ctrl->put & ~3u))
```

Comment:
Exact same check is repeated 4 times in one file. Consider adding a method rsx_thread::is_idle() or rsx_thread::is_FIFO_idle() for reusability.